druid-shell/examples/shello.rs

```diff
@@ -133,8 +141,17 @@ fn main() {
     builder.set_handler(Box::new(HelloState::default()));
     builder.set_title("Hello example");
     builder.set_menu(menubar);
-    let window = builder.build().unwrap();
 
-    window.show();
+    // TODO(x11/architecture): super hacky way to connect the XWindow to the RunLoop. Better way to do it?
```

Comment:
ditto this, I could've sworn I deleted this?

Reply:
Fixed in my branch :)